paasta_tools/marathon_tools.py

```diff
@@ -993,3 +1015,47 @@ def kill_task(client, app_id, task_id, scale):
             return []
         else:
             raise
+
+
+def get_instances_from_zookeeper(service, instance):
+    with ZookeeperPool() as zookeeper_client:
+        (instances, _) = zookeeper_client.get('/autoscaling/%s/%s/instances' % (service, instance))
+        return int(instances)
+
+
+def update_instances_for_marathon_service(service, instance, delta):
+    zookeeper_path = '/autoscaling/%s/%s/instances' % (service, instance)
+    marathon_config = load_marathon_service_config(
+        service=service,
+        instance=instance,
+        cluster=load_system_paasta_config().get_cluster(),
+        load_deployments=False,
+    )
+    with ZookeeperPool() as zookeeper_client:
+        instances = min(
+            marathon_config.get_max_instances(),
+            max(marathon_config.get_min_instances(), marathon_config.get_instances() + delta))
+        zookeeper_client.ensure_path(zookeeper_path)
+        zookeeper_client.set(zookeeper_path, instances)
+        return instances
+
+
+class ZookeeperPool(object):
+    counter = 0
+    zk = None
+
+    @classmethod
+    def __enter__(cls):
+        if cls.zk is None:
+            cls.zk = KazooClient(hosts=load_system_paasta_config().get_zk_hosts(), read_only=True)
```

Comment:
@solarkennedy AFAICT you just add the chroot to the end of the host list

For example `zk://10.40.5.5:22181,10.40.5.6:22181,10.40.1.17:22181/mesos-mesosstage`